packages/ui/src/components/IdenticonBadge.tsx

```diff
@@ -90,7 +95,14 @@ const BadgeStyled = styled(Badge)<{ badgeType: AccountBadge; sideBadge: boolean
         position: relative;
         transform: unset;
       `}
-  }
+
+    ${({ size }) =>
```

Comment:
Can we also have the "pure" written a little bit bigger
![image](https://github.com/ChainSafe/Multix/assets/33178835/a5c1e3f7-5be4-4e18-a03f-309f1ed172c4)
the figma has `font-size: 0.96431rem; `

Reply:
done